index.js

```diff
@@ -59,6 +72,36 @@ function getDuplicates(cwd, ...patterns) {
   return duplicates;
 }
 
+async function getPackageVersion({ pathName, ...rest }) {
+  const pkg = await readFile(path.join(pathName, 'package.json'), 'utf8');
+  const { version } = JSON.parse(pkg);
+  return { pathName, version, ...rest };
+}
+
+async function getDuplicatesDetails(cwd, ...patterns) {
```

Comment:
Hm, I don't like the name so much, how about `getDuplicatesWithVersion` or `getDuplicatesAndVersions`?

Reply:
I went with the more open name, because the approach enables us to include more data from the package's `package.json` files than just the version. In case we need to do this, we wouldn't have to introduce another breaking change.